protocol/communities/manager.go

```diff
@@ -5,6 +5,7 @@ import (
 	"database/sql"
 	"fmt"
 	"os"
+	"sort"
```

Comment:
This comment here is just to ensure this file is being reviewed, as github collapses it due to the amount of changes.